application/features/analytics.feature

```diff
@@ -61,4 +61,41 @@ Feature: User
         And that user has an unverified u2f mfa record
       And I create a new user
     When I get the count of active users with a verified u2f mfa
-    Then the count of active users with a verified u2f mfa should be 1
\ No newline at end of file
+    Then the count of active users with a verified u2f mfa should be 1
+
+  Scenario: Get a count of active users with a password
+    Given that no mfas or users exist
+      And I create a new user with a password
+      And I create a new user with a password
+      And I create a new user without a password
+      And I create a new user without a password
+    When I get the count of active users with a password
+    Then the count of active users with a password should be 2
+
+  Scenario: Get the average number of mfas per active user with a verified mfa
+    Given that no mfas or users exist
+      And I create a new user
+        And that user has a backup code mfa record
+      And I create a new user
+        And that user has a verified totp mfa record
+        And that user has a backup code mfa record
+      And I create a new user
+        And that user has a verified u2f mfa record
+        And that user has another verified u2f mfa record
+        And that user has a backup code mfa record
+      And I create a new user
+        And that user has an unverified u2f mfa record
+      And I create a new inactive user
+        And that user has a backup code mfa record
+      And I create a new user
+    When I get the average number of mfas per active user with mfas
+    Then the average number of mfas per active user with mfas should be 2
+
+  Scenario: Get a count of active users with require mfa
+    Given that no mfas or users exist
+      And I create a new user with require mfa
+      And I create a new user with require mfa
+      And I create a new user
+      And I create a new user
```

Comment:
As above, it may be worth having a different number of each type of user, to make sure the logic is counting the correct set of users.